index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio Mobile First</title>
+</head>
+
+<body>
+    <main id="main-section">
+
+        <nav class="navbar">
+            <a href="#">Nasir</a>
+            <img src="images/Union.png" alt="button">
+        </nav>
+
+        <section class="about">
+
+
+            <h1>Hi, I am Nasir.<br>
+                Glad to see <br>
+                you!</h1>
+
+            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my
+                work and
+                experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+            </p>
+
+            <a href="#">LET'S CONNECT</a>
+            <ul class="socialmedia">
+                <li><img src="images/twitter.png" alt="twitterlogo"></li>
+                <li><img src="images/linkedIn.png" alt="linkedinlogo"></li>
+                <li><img src="images/medium.png" alt="mediumlogo"></li>
+                <li><img src="images/github.png" alt="githublogo"></li>
+                <li><img src="images/angellist.png" alt="angellistlogo"></li>
+            </ul>
```

Comment:
- Great job for using `ul` and `li` for social icons 👍 I think it would be better if you wrap each social icon with an `a` tag because Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using `href`.👌

Reply:
Alright 👏 Thanks for the feedback on here as well.